### ISSUE TITLE
Android version

### ISSUE THREAD

**Issue by GustawOhler** [crazycodeboy/react-native-splash-screen#292]
What is version of Android needed to properly use this framework? Because on one device I have 5.1.1 and it doesn't work, just displays dark screen.

**Comment by kristfal** [comment]
I'm seeing the same issue on _one_ of our test devices. A 5.0.1 Xiaomi Redmi Note 3. We've been running this down to 4.4.4 on a Samsung S III with no issue whatsoever. Not sure what causes this.

**Comment by GustawOhler** [comment]
Yeah, the worst is I can't test my app sometimes when I have this bug. I mean on some devices the splash screen just doesn't work and it goes white, then displays app and on the others it goes dark and just doesn't go away so i can't do anything then... Somebody has idea how to get past this?
